atlas-irap/tag

```diff
@@ -0,0 +1 @@
+v0.0.1
```

Comment:
This isn't how container image tags, which is part of what's upsetting the CI, have a loot at a couple of the other container directories to see what I mean.